src/_includes/head.html

```diff
@@ -17,14 +19,24 @@
   <meta name="twitter:card" content="summary" />
   <meta name="twitter:site" content="@dart_lang" />
   <meta name="twitter:title" content="{{ page.title }}" />
-  <meta name="twitter:description" content="{% if page.description | strip_html | strip_newlines | truncate: 160 %}{{ page.description }}{% else if page.excerpt %}{{ page.excerpt | strip_html | strip_newlines | truncate: 160 }}{% else %}{{ site.description | strip_html | strip_newlines | truncate: 160 }}{% endif %}" />
+  <meta name="twitter:description" content="{{desc}}" />
 
   <!-- Open Graph -->
   <meta property="og:title" content="{{ page.title }}" />
-  <meta property="og:description" content="{% if page.description | strip_html | strip_newlines | truncate: 160 %}{{ page.description }}{% else if page.excerpt %}{{ page.excerpt | strip_html | strip_newlines | truncate: 160 }}{% else %}{{ site.description | strip_html | strip_newlines | truncate: 160 }}{% endif %}" />
-  <meta property="og:type" content="website" />
-  <meta property="og:url" content="{{ site.url }}{{ page.url }}" />
-  <meta property="og:image" content="{% if page.image %}{{ page.image }}{% else %}{{ site.url }}{% asset_path 'dart-logo-wordmark.png' %}{% endif %}" />
+  <meta property="og:description" content="{{desc}}" />
+  <meta property="og:url" content="{{ page.url | absolute_url }}" />{%
+    assign og_image_asset = page.image_asset | default: layout.image_asset | default: 'dart-logo-for-shares.png' | asset_path %}{%
```

Comment:
Ah, OK. I must've been looking at a slightly older version of the webdev head.html file.